src/index.html

```diff
@@ -8,6 +8,111 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="big_star_container">
+      <div class="star_container stars--0">
```

Comment:
```suggestion
      <div class="stars stars--0">
```
![image](https://user-images.githubusercontent.com/96409563/217354837-c6e5011d-1603-49d5-ba3b-95ef5eceb5e0.png)